app/frontend/css/cdo-task-list.scss

```diff
@@ -0,0 +1,56 @@
+.defra {
```

Comment:
Is there not a standard GDS component we can use and lever the styling on it, such as table or summary-card?